tensorrt_llm/_torch/custom_ops/cute_dsl_custom_ops.py

```
@@ -2789,3 +2793,308 @@ def _(
+                    sf_k,
+                    1,  # batch
+                    group_size,
+                    a_ptr,
+                    b_ptr,
+                    a_sf_ptr,
+                    b_sf_ptr,
+                    c_ptr,
+                    group_offset_cute_tensor,
+                    max_active_clusters,
+                    stream,
+                    options=f"--opt-level 2 --enable-tvm-ffi"
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)